lib/Launcher.js

```diff
@@ -83,26 +83,32 @@ class Launcher {
 
     // Cleanup as processes exit.
     const listeners = [
-      helper.addEventListener(process, 'exit', killChromeAndCleanup),
-      helper.addEventListener(chromeProcess, 'exit', killChromeAndCleanup),
+      helper.addEventListener(process, 'exit', killChrome),
```

Comment:
the `helper.removeEventListeners` is not needed any more. Could you please drop the `listeners` altogether and use the `once` methods?
```js
  chromeProcess.once('close', () => removeSync(userDataDir));
  process.once('exit', killChrome);
  if (options.handleSIGINT !== false)
    process.once('SIGINT', killChrome);
```